registry-requirements.md

```diff
@@ -44,24 +43,22 @@ To choose a mechanism and the rules, we document our expectations below:
+  - DISCUSS: Clarify what happens when two ecosystems like OCF and LwM2M both use CoAP binding. The initial thinking is to register them as separate entries but clarify what they use from CoAP binding. A layered registry can be thought of as similar to language tags with `en` extending to `en-us` and `en-uk`, where the tags and entries are different but the association with `en` is expressed in the id. In our case, this would be `coap-ocf` and `coap-lwm2m`. A higher-level binding SHOULD NOT override or conflict with a lower-level binding, and this should be verified by the custodian, e.g., `cov:method` in CoAP binding should not be turned into `cov:operation` in the higher-level binding. The namespace (prefix and its values) defined in a binding CANNOT be redefined in any other binding.
+- In a TD, a binding SHOULD be identifiable by the terms in a form such as `href`, `contentType`, `subprotocol`, or connection information found in the root of the TD.
   - Reasons Behind the Requirement:
     - This avoids conflicts that are mentioned in the previous requirement
   - TODO: These terms should be refined based on the additions/changes to the TD 2.0 mechanism. E.g. introducing a `protocol` term, or restrictions on URI scheme and `subprotocol` combination, data mapping etc.
-  - TODO: We need to clarify whether the URI scheme and the media type MUST be registered in IANA first.
+  - DISCUSS: We need to clarify whether the URI scheme and the media type MUST be registered in IANA first.
     - Provisional registration could reduce the overhead. Any new conflicting ones would bring up a discussion but it can still result in "our" provisional getting demoted.
     - Pros: more stable. Cons: More overhead and work
-  - TODO: How to identify a higher-level binding in the form
+  - DISCUSS: How to identify a higher-level binding in the form
 - A binding that uses a protocol MUST map at least one WoT operation (`op` keyword values such as `readproperty`) to a protocol message and vice versa
   - Reasons Behind the Requirement:
     - Otherwise, it is not binding the protocol to WoT and cannot be useful.
 - A binding that uses a serialization format via the `contentType` keyword, MUST mention how the Data Schema terms should be used to describe the messages.
   - Reasons Behind the Requirement:
     - Avoid submission of a binding like "XML Binding" that only says "Use `contentType:application/xml` and nothing more. That alone would not be enough to serialize correct messages based on the data schema.
   - TODO: We will need additional mechanisms (including vocabulary terms) to ensure that it is possible to use other media types.
-- TODO: Explain that there are no categories
-- TODO: Analyze <https://github.com/w3c/ttwg/tree/main/boilerplate/registry>
```

Comment:
These two points are already discussed. We agreed that there will be no categories and the analysis of TTWG boilerplate is done already.